.env.versions

```diff
@@ -16,7 +16,7 @@ NODEJS_VERSION=24.10.0
 NUGET_INSPECTOR_VERSION=0.9.12
 PHP_VERSION=8.3
 PIP_VERSION=25.2.0
-PYENV_GIT_TAG=v2.6.3
+PYENV_VERSION=2.6.3
```

Comment:
Why don't we stick to `PYENV_GIT_TAG` as the `ARG` name, though? It should be no problem to name `ENV` and `ARG` keys the same, as [that's a common work-around](https://docs.docker.com/build/building/variables/#env-usage-example).

Reply:
Hm. I tried to read up on `ARG` semantics, but didn't find the information which proofs that the previous implementation does not work. Would you know @sschuberth the reason why it should not work?

Reply:
> So does the `PYENV_VERSION` environment variable have a pre-defined meaning maybe?

still fails, so I'll drop this PR.